torch/csrc/jit/serialization/unpickler.cpp

```diff
@@ -522,6 +520,8 @@ void Unpickler::readGlobal(
         elem_type = TensorType::get();
       } else if (class_name == "build_doublelist") {
         elem_type = FloatType::get();
+      } else if (class_name == "build_complexdoublelist") {
+        elem_type = ComplexDoubleType::get();
```

Comment:
Do we have tests for this?

Reply:
good catch! this change shouldn't be here because complex lists are not supported yet